Worktez/worktez
# Dev angular

### Functionality:
Added changes in the UI of the task detail page

### Solution:
added a scrollbar to the comment section 
brought the comment section to the bottom and made it static
Changed the text cursors to pointers 
Changed few text cursors to default cursors
Increased the height of the container as per the UI provided
Added css to the task detail heading written at the top
Made the borders thinner

### Risk level:
- [*] low

### How to test:
Check the comment section of the task details page 
check the position of the comment box
check the cursors
check the size of the task detail container 
check the color of the and the borders
